### PR TITLE
Half-sprint PR

### PR DESCRIPTION
**Pull request by jimmynotjim**
Merging mid-sprint because broken tests keep bringing the party down

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # cfgov-refresh
 
+[![Build Status](https://travis-ci.org/cfpb/cfgov-refresh.png?branch=flapjack)](https://travis-ci.org/cfpb/cfgov-refresh?branch=flapjack)
+[![Coverage Status](https://coveralls.io/repos/cfpb/cfgov-refresh/badge.png?branch=flapjack)](https://coveralls.io/r/cfpb/cfgov-refresh?branch=flapjack)
```
Comment (anselmbradford) [inline README.md]: Probably should remove coveralls badge for now till coveralls can pull in a different branch than master. Code Climate also has a coverage badge that comes up with unknown. If we can use the Code Climate one then it reduces the services.

File: src/static/css/skip-nav.less [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    position: absolute;
+    top: unit(-@grid_gutter-width / @base-font-size-px, em);
+    left: unit(@grid_gutter-width / @base-font-size-px, em);
+    background: transparent;
+    transition: transform 1s ease, background .5s linear;
+    z-index: 10;
+}
+
+#skip-nav:focus {
+    .btn;
```
Comment (anselmbradford) [inline src/static/css/skip-nav.less]: Is there a difference in using `.btn` vs `.btn()` in Less?

Comment (jimmynotjim) [inline src/static/css/skip-nav.less]: There isn't when using it if you're not passing any params. The main difference is when defining mixins, if you don't add parens, it'll treat it like a class and include it in your final style sheet. Ex.

```
Gets treated like both a class and mixin, thus printing this rule
.foo {
   display: block;
}

// Only gets treated like a mixin, so it's not printed as a rule
.bar() {
  display: block;
}
```

File: src/static/js/modules/util/breakpoint-state.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+   Get Breakpoint State
+   ========================================================================== */
+
+'use strict';
+
+var _breakpointsConfig = require( '../../config/breakpoints-config' );
+var _getViewportDimensions = require( './get-viewport-dimensions' )
+                             .getViewportDimensions;
+
+function _inBreakpointRange( breakpointRange, width ) {
```
Comment (anselmbradford) [inline src/static/js/modules/util/breakpoint-state.js]: Would be helpful to add jsdocs to this method.

File: src/static/js/modules/util/breakpoint-state.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  var min = breakpointRange.min || 0;
+  var max = breakpointRange.max || Number.POSITIVE_INFINITY;
+
+  return min <= width && width <= max;
+}
+
+/**
+ * @returns {object} An object literal with Boolean
+ * isMobile, isTablet, isDesktop, isWall, isSmall properties.
+ * @param {integer} width Current window width.
```
Comment (anselmbradford) [inline src/static/js/modules/util/breakpoint-state.js]: Place `@param` before `@returns` to follow execution order of function.

File: src/static/js/modules/util/breakpoint-state.js (hunk 3 above)
Comment (sebworks) [inline src/static/js/modules/util/breakpoint-state.js]: Yep, will add it.

**Comment by sebworks** [comment]
:+1:

**Comment by anselmbradford** [comment]
:+1:

File: src/static/css/skip-nav.less (hunk 2 above)
Comment (anselmbradford) [inline src/static/css/skip-nav.less]: Thanks! Makes sense

**Pull request merged**